apps/web-giddh/src/app/theme/ngx-date-range-picker/ngx-daterangepicker.component.scss

```diff
@@ -492,7 +602,7 @@ $input-height: 3rem !default;
     /* ranges */
     .show-ranges {
         .drp-calendar.left {
-            border-left: 1px solid #ddd;
+            border-left: 1px solid$gainsboro-light;
```

Comment:
1px solid$gainsboro-light; add some space before color var , may be it will be not apply

Reply:
@shalinee-walkover done